net-tools/netwatch/Cargo.toml

```diff
@@ -15,28 +15,34 @@ workspace = true
 [dependencies]
 anyhow = { version = "1" }
 bytes = "1.7"
-futures-lite = "2.3"
-futures-sink = "0.3.25"
-futures-util = "0.3.25"
+futures-lite = "2.5"
+futures-sink = "0.3"
+futures-util = "0.3"
 libc = "0.2.139"
-netdev = "0.30.0"
+netdev = "0.31.0"
 once_cell = "1.18.0"
 socket2 = "0.5.3"
-thiserror = "1"
+thiserror = "2"
 time = "0.3.20"
 tokio = { version = "1", features = ["io-util", "macros", "sync", "rt", "net", "fs", "io-std", "signal", "process", "time"] }
 tokio-util = { version = "0.7", features = ["rt"] }
 tracing = "0.1"
 
-[target.'cfg(any(target_os = "linux", target_os = "android"))'.dependencies]
+[target.'cfg(all(target_os = "linux", not(target_os = "android")))'.dependencies]
 netlink-packet-core = "0.7.0"
-netlink-packet-route = "0.17.0"
-netlink-sys = "0.8.5"
-rtnetlink = "0.13.0"
+netlink-packet-route = "0.19" # 0.20/21 is blocked on rtnetlink bumping its dependency
+netlink-sys = "0.8.6"
+rtnetlink = "=0.14.1" # pinned because of https://github.com/rust-netlink/rtnetlink/issues/83
+
+[target.'cfg(target_os = "android")'.dependencies]
```

Comment:
This is the ugly bit for android but is necessary right now unless we want to keep linux down to the same version it was because of android.

Reply:
Also need a way to track this too.